### PR TITLE
add cbr support

### PR DESCRIPTION
**Pull request by Aayush-Abhyarthi**
### Description

This PR adds cbr support in scc workload protection module

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

This PR adds the cbr support in the scc workload protection module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -41,3 +41,34 @@ resource "ibm_resource_tag" "scc_wp_access_tag" {
...
+# Context Based Restrictions
+##############################################################################
+module "cbr_rule" {
+  count            = length(var.cbr_rules) > 0 ? length(var.cbr_rules) : 0
+  source           = "terraform-ibm-modules/cbr/ibm//modules/cbr-rule-module"
+  version          = "1.24.0"
+  rule_description = var.cbr_rules[count.index].description
+  enforcement_mode = var.cbr_rules[count.index].enforcement_mode
+  rule_contexts    = var.cbr_rules[count.index].rule_contexts
+  resources = [{
```
Comment (akocbek) [inline main.tf:55]: shall we add support for tags as well?
similar as we have for cos buckets https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/b9cc0f04153b289e582f80175d9bc93d648189e3/main.tf#L376

**Review by akocbek — CHANGES_REQUESTED** [review]

**Comment by ocofaigh** [comment]
@Aayush-Abhyarthi @akocbek PR has been open for 3 weeks - are we all good here?

**Comment by Aayush-Abhyarthi** [comment]
> @Aayush-Abhyarthi @akocbek PR has been open for 3 weeks - are we all good here?

Its ok from my side. Was waiting for the re-review of the proposed changes

**Comment by ocofaigh** [comment]
Please reach out to @akocbek for a re-review so as it looks like he originally reviewed it. Thanks

**Review by akocbek — APPROVED** [review]

**Pull request merged**